syntax = "proto3";

message uartMessage{
  string mac = 1;
  string ssid = 2;
  uint32 channel = 3;
  int32 rssi = 4;
  int32 authmode = 5;
  int32 pairwise_cipher = 6;
  int32 groupwise_cipher = 7;
  string country = 8;
}